src/User/UserRepository.php

```diff
@@ -11,22 +11,28 @@ public function __construct(
     ) {
     }
 
-    public function getOrCreate(string $githubId, string $name): User
+    public function getOrCreate(string $githubId, string $ghName): User
     {
         $userData = $this->db->fetchAssociative('SELECT * FROM users WHERE githubId = ?', [$githubId]);
 
         if ($userData) {
-            return new User((int) $userData['id'], $githubId, (string) $userData['name']);
+            $id = (int) $userData['id'];
+
+            if ($ghName !== (string) $userData['name']) {
+                $this->db->update('users', ['name' => $ghName], ['id' => $id]);
+            }
```

Comment:
This is kind of awkward. From what I gather, we could simply remove the `name` column from `users`, as I don't see it used anywhere.

Reply:
😅  that's very much possible, the initial plan may have included some ideas like "let's build a page in the future where people can update their profiles"

🤷  we don't _have_ to fix this right now.